src/pip/_internal/operations/prepare.py

```diff
@@ -126,25 +126,31 @@ def prep_for_dist(self, finder, build_isolation):
         build_requirements, isolate = self.req.get_pep_518_info()
         should_isolate = build_isolation and isolate
 
-        if 'setuptools' not in build_requirements:
+        minimum_requirements = ('setuptools', 'wheel')
+        missing_requirements = set(minimum_requirements) - set(
+            pkg_resources.Requirement(r).key
+            for r in build_requirements
+        )
+        if missing_requirements:
+            def format_reqs(rs):
+                return ' and '.join(map(repr, sorted(rs)))
             logger.warning(
-                "%s does not include 'setuptools' as a buildtime requirement "
-                "in its pyproject.toml.", self.req.name,
+                "%s does not include %s as build time requirements in its "
+                "pyproject.toml.", self.req, format_reqs(missing_requirements)
```

Comment:
nit: plural "requirements" looks odd if there's only one missing requirement. Maybe reword? Something like "Missing buildtime requirements in pyproject.toml for %s: %s"?